### ISSUE TITLE
Rollup 2.36.* produces invalid output

### ISSUE THREAD

**Issue by IronGeek** [rollup/rollup#3924]
- Rollup Version: >=2.36.0 (including the latest 2.36.1)
- Operating System (or Browser): any
- Node Version (if applicable):
- Link to reproduction (IMPORTANT, read below): [repl](https://rollupjs.org/repl/?version=2.36.1&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMGlzV2ViJTIwJTdEJTIwZnJvbSUyMCcuJTJGcGxhdGZvcm0uanMnJTNCJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlN0QlMkMlN0IlMjJuYW1lJTIyJTNBJTIycGxhdGZvcm0uanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyJTJGKi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLSU1Q24lMjAqJTIwJTIwQ29weXJpZ2h0JTIwKGMpJTIwTWljcm9zb2Z0JTIwQ29ycG9yYXRpb24uJTIwQWxsJTIwcmlnaHRzJTIwcmVzZXJ2ZWQuJTVDbiUyMColMjAlMjBMaWNlbnNlZCUyMHVuZGVyJTIwdGhlJTIwTUlUJTIwTGljZW5zZS4lMjBTZWUlMjBMaWNlbnNlLnR4dCUyMGluJTIwdGhlJTIwcHJvamVjdCUyMHJvb3QlMjBmb3IlMjBsaWNlbnNlJTIwaW5mb3JtYXRpb24uJTVDbiUyMCotLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLSolMkYlNUNuY29uc3QlMjBMQU5HVUFHRV9ERUZBVUxUJTIwJTNEJTIwJ2VuJyUzQiU1Q25sZXQlMjBfaXNXaW5kb3dzJTIwJTNEJTIwZmFsc2UlM0IlNUNubGV0JTIwX2lzTWFjaW50b3NoJTIwJTNEJTIwZmFsc2UlM0IlNUNubGV0JTIwX2lzTGludXglMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfaXNOYXRpdmUlMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfaXNXZWIlMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfaXNJT1MlMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfbG9jYWxlJTIwJTNEJTIwdW5kZWZpbmVkJTNCJTVDbmxldCUyMF9sYW5ndWFnZSUyMCUzRCUyMExBTkdVQUdFX0RFRkFVTFQlM0IlNUNubGV0JTIwX3RyYW5zbGF0aW9uc0NvbmZpZ0ZpbGUlMjAlM0QlMjB1bmRlZmluZWQlM0IlNUNubGV0JTIwX3VzZXJBZ2VudCUyMCUzRCUyMHVuZGVmaW5lZCUzQiU1Q25jb25zdCUyMGlzRWxlY3Ryb25SZW5kZXJlciUyMCUzRCUyMCh0eXBlb2YlMjBwcm9jZXNzJTIwISUzRCUzRCUyMCd1bmRlZmluZWQnJTIwJTI2JTI2JTIwdHlwZW9mJTIwcHJvY2Vzcy52ZXJzaW9ucyUyMCElM0QlM0QlMjAndW5kZWZpbmVkJyUyMCUyNiUyNiUyMHR5cGVvZiUyMHByb2Nlc3MudmVyc2lvbnMuZWxlY3Ryb24lMjAhJTNEJTNEJTIwJ3VuZGVmaW5lZCclMjAlMjYlMjYlMjBwcm9jZXNzLnR5cGUlMjAlM0QlM0QlM0QlMjAncmVuZGVyZXInKSUzQiU1Q24lMkYlMkYlMjBPUyUyMGRldGVjdGlvbiU1Q25pZiUyMCh0eXBlb2YlMjBuYXZpZ2F0b3IlMjAlM0QlM0QlM0QlMjAnb2JqZWN0JyUyMCUyNiUyNiUyMCFpc0VsZWN0cm9uUmVuZGVyZXIpJTIwJTdCJTVDbiUyMCUyMCUyMCUyMF91c2VyQWdlbnQlMjAlM0QlMjBuYXZpZ2F0b3IudXNlckFnZW50JTNCJTVDbiUyMCUyMCUyMCUyMF9pc1dpbmRvd3MlMjAlM0QlMjBfdXNlckFnZW50LmluZGV4T2YoJ1dpbmRvd3MnKSUyMCUzRSUzRCUyMDAlM0IlNUNuJTIwJTIwJTIwJTIwX2lzTWFjaW50b3NoJTIwJTNEJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdNYWNpbnRvc2gnKSUyMCUzRSUzRCUyMDAlM0IlNUNuJTIwJTIwJTIwJTIwX2lzSU9TJTIwJTNEJTIwKF91c2VyQWdlbnQuaW5kZXhPZignTWFjaW50b3NoJyklMjAlM0UlM0QlMjAwJTIwJTdDJTdDJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdpUGFkJyklMjAlM0UlM0QlMjAwJTIwJTdDJTdDJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdpUGhvbmUnKSUyMCUzRSUzRCUyMDApJTIwJTI2JTI2JTIwISFuYXZpZ2F0b3IubWF4VG91Y2hQb2ludHMlMjAlMjYlMjYlMjBuYXZpZ2F0b3IubWF4VG91Y2hQb2ludHMlMjAlM0UlMjAwJTNCJTVDbiUyMCUyMCUyMCUyMF9pc0xpbnV4JTIwJTNEJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdMaW51eCcpJTIwJTNFJTNEJTIwMCUzQiU1Q24lMjAlMjAlMjAlMjBfaXNXZWIlMjAlM0QlMjB0cnVlJTNCJTVDbiUyMCUyMCUyMCUyMF9sb2NhbGUlMjAlM0QlMjBuYXZpZ2F0b3IubGFuZ3VhZ2UlM0IlNUNuJTIwJTIwJTIwJTIwX2xhbmd1YWdlJTIwJTNEJTIwX2xvY2FsZSUzQiU1Q24lN0QlNUNuZWxzZSUyMGlmJTIwKHR5cGVvZiUyMHByb2Nlc3MlMjAlM0QlM0QlM0QlMjAnb2JqZWN0JyklMjAlN0IlNUNuJTIwJTIwJTIwJTIwX2lzV2luZG93cyUyMCUzRCUyMChwcm9jZXNzLnBsYXRmb3JtJTIwJTNEJTNEJTNEJTIwJ3dpbjMyJyklM0IlNUNuJTIwJTIwJTIwJTIwX2lzTWFjaW50b3NoJTIwJTNEJTIwKHByb2Nlc3MucGxhdGZvcm0lMjAlM0QlM0QlM0QlMjAnZGFyd2luJyklM0IlNUNuJTIwJTIwJTIwJTIwX2lzTGludXglMjAlM0QlMjAocHJvY2Vzcy5wbGF0Zm9ybSUyMCUzRCUzRCUzRCUyMCdsaW51eCcpJTNCJTVDbiUyMCUyMCUyMCUyMF9sb2NhbGUlMjAlM0QlMjBMQU5HVUFHRV9ERUZBVUxUJTNCJTVDbiUyMCUyMCUyMCUyMF9sYW5ndWFnZSUyMCUzRCUyMExBTkdVQUdFX0RFRkFVTFQlM0IlMjAlMjAlMjAlMjAlNUNuJTIwJTIwJTIwJTIwX2lzTmF0aXZlJTIwJTNEJTIwdHJ1ZSUzQiU1Q24lN0QlNUNubGV0JTIwX3BsYXRmb3JtJTIwJTNEJTIwMCUyMCUyRiolMjBXZWIlMjAqJTJGJTNCJTVDbmlmJTIwKF9pc01hY2ludG9zaCklMjAlN0IlNUNuJTIwJTIwJTIwJTIwX3BsYXRmb3JtJTIwJTNEJTIwMSUyMCUyRiolMjBNYWMlMjAqJTJGJTNCJTVDbiU3RCU1Q25lbHNlJTIwaWYlMjAoX2lzV2luZG93cyklMjAlN0IlNUNuJTIwJTIwJTIwJTIwX3BsYXRmb3JtJTIwJTNEJTIwMyUyMCUyRiolMjBXaW5kb3dzJTIwKiUyRiUzQiU1Q24lN0QlNUNuZWxzZSUyMGlmJTIwKF9pc0xpbnV4KSUyMCU3QiU1Q24lMjAlMjAlMjAlMjBfcGxhdGZvcm0lMjAlM0QlMjAyJTIwJTJGKiUyMExpbnV4JTIwKiUyRiUzQiU1Q24lN0QlNUNuZXhwb3J0JTIwY29uc3QlMjBpc1dpbmRvd3MlMjAlM0QlMjBfaXNXaW5kb3dzJTNCJTVDbmV4cG9ydCUyMGNvbnN0JTIwaXNNYWNpbnRvc2glMjAlM0QlMjBfaXNNYWNpbnRvc2glM0IlNUNuZXhwb3J0JTIwY29uc3QlMjBpc0xpbnV4JTIwJTNEJTIwX2lzTGludXglM0IlNUNuZXhwb3J0JTIwY29uc3QlMjBpc05hdGl2ZSUyMCUzRCUyMF9pc05hdGl2ZSUzQiU1Q25leHBvcnQlMjBjb25zdCUyMGlzV2ViJTIwJTNEJTIwX2lzV2ViJTNCJTVDbmV4cG9ydCUyMGNvbnN0JTIwaXNJT1MlMjAlM0QlMjBfaXNJT1MlM0IlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIydW1kJTIyJTJDJTIybmFtZSUyMiUzQSUyMm15QnVuZGxlJTIyJTJDJTIyYW1kJTIyJTNBJTdCJTIyaWQlMjIlM0ElMjIlMjIlN0QlMkMlMjJnbG9iYWxzJTIyJTNBJTdCJTdEJTdEJTJDJTIyZXhhbXBsZSUyMiUzQW51bGwlN0Q=)

Rollup >=2.36.0 produces invalid output when used to bundle the following code:
```javascript
const LANGUAGE_DEFAULT = 'en';
let _isWindows = false;
let _isMacintosh = false;
let _isLinux = false;
let _isNative = false;
let _isWeb = false;
let _isIOS = false;
let _locale = undefined;
let _language = LANGUAGE_DEFAULT;
let _translationsConfigFile = undefined;
let _userAgent = undefined;
const isElectronRenderer = (typeof process !== 'undefined' && typeof process.versions !== 'undefined' && typeof process.versions.electron !== 'undefined' && process.type === 'renderer');
// OS detection
if (typeof navigator === 'object' && !isElectronRenderer) {
    _userAgent = navigator.userAgent;
    _isWindows = _userAgent.indexOf('Windows') >= 0;
    _isMacintosh = _userAgent.indexOf('Macintosh') >= 0;
    _isIOS = (_userAgent.indexOf('Macintosh') >= 0 || _userAgent.indexOf('iPad') >= 0 || _userAgent.indexOf('iPhone') >= 0) && !!navigator.maxTouchPoints && navigator.maxTouchPoints > 0;
    _isLinux = _userAgent.indexOf('Linux') >= 0;
    _isWeb = true;
    _locale = navigator.language;
    _language = _locale;
}
else if (typeof process === 'object') {
    _isWindows = (process.platform === 'win32');
    _isMacintosh = (process.platform === 'darwin');
    _isLinux = (process.platform === 'linux');
    _locale = LANGUAGE_DEFAULT;
    _language = LANGUAGE_DEFAULT;    
    _isNative = true;
}
...
```
### Expected Behavior

Produce valid output (rollup v2.35.1 produces this valid result)

```javascript
...
if (typeof navigator === 'object' && !isElectronRenderer) {
  _userAgent = navigator.userAgent;
  _isWindows = _userAgent.indexOf('Windows') >= 0;
  _isMacintosh = _userAgent.indexOf('Macintosh') >= 0;
  _isIOS = (_userAgent.indexOf('Macintosh') >= 0 || _userAgent.indexOf('iPad') >= 0 || _userAgent.indexOf('iPhone') >= 0) && !!navigator.maxTouchPoints && navigator.maxTouchPoints > 0;
  _isLinux = _userAgent.indexOf('Linux') >= 0;
}
else if (typeof process === 'object') {
  _isWindows = (process.platform === 'win32');
  _isMacintosh = (process.platform === 'darwin');
  _isLinux = (process.platform === 'linux');
}
...
```

### Actual Behavior

Produce invalid output:
```javascript
...
if (typeof navigator === 'object' && !isElectronRenderer) {
  _userAgent = navigator.userAgent;
  _userAgent.indexOf('Windows') >= 0;
  _userAgent.indexOf('Macintosh') >= 0;
  (_userAgent.indexOf('Macintosh') >= 0 || _userAgent.indexOf('iPad') >= 0 || _userAgent.indexOf('iPhone') >= 0) && !!navigator.maxTouchPoints && navigator.maxTouchPoints > 0;
  _userAgent.indexOf('Linux') >= 0;
}
else if (typeof process === 'object') {
  process.platform === 'win32');
  process.platform === 'darwin');
  process.platform === 'linux');
}
...
```

**Comment by lukastaegert** [comment]
Could you please clarify further what is the invalid part? Wrong parentheses? Code removed that is essential?

**Comment by IronGeek** [comment]
Yes, part of the code is removed which resulted in syntax error. 
Result in repro [here](https://rollupjs.org/repl/?version=2.36.1&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMGlzV2ViJTIwJTdEJTIwZnJvbSUyMCcuJTJGcGxhdGZvcm0uanMnJTNCJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlN0QlMkMlN0IlMjJuYW1lJTIyJTNBJTIycGxhdGZvcm0uanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyJTJGKi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLSU1Q24lMjAqJTIwJTIwQ29weXJpZ2h0JTIwKGMpJTIwTWljcm9zb2Z0JTIwQ29ycG9yYXRpb24uJTIwQWxsJTIwcmlnaHRzJTIwcmVzZXJ2ZWQuJTVDbiUyMColMjAlMjBMaWNlbnNlZCUyMHVuZGVyJTIwdGhlJTIwTUlUJTIwTGljZW5zZS4lMjBTZWUlMjBMaWNlbnNlLnR4dCUyMGluJTIwdGhlJTIwcHJvamVjdCUyMHJvb3QlMjBmb3IlMjBsaWNlbnNlJTIwaW5mb3JtYXRpb24uJTVDbiUyMCotLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLSolMkYlNUNuY29uc3QlMjBMQU5HVUFHRV9ERUZBVUxUJTIwJTNEJTIwJ2VuJyUzQiU1Q25sZXQlMjBfaXNXaW5kb3dzJTIwJTNEJTIwZmFsc2UlM0IlNUNubGV0JTIwX2lzTWFjaW50b3NoJTIwJTNEJTIwZmFsc2UlM0IlNUNubGV0JTIwX2lzTGludXglMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfaXNOYXRpdmUlMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfaXNXZWIlMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfaXNJT1MlMjAlM0QlMjBmYWxzZSUzQiU1Q25sZXQlMjBfbG9jYWxlJTIwJTNEJTIwdW5kZWZpbmVkJTNCJTVDbmxldCUyMF9sYW5ndWFnZSUyMCUzRCUyMExBTkdVQUdFX0RFRkFVTFQlM0IlNUNubGV0JTIwX3RyYW5zbGF0aW9uc0NvbmZpZ0ZpbGUlMjAlM0QlMjB1bmRlZmluZWQlM0IlNUNubGV0JTIwX3VzZXJBZ2VudCUyMCUzRCUyMHVuZGVmaW5lZCUzQiU1Q25jb25zdCUyMGlzRWxlY3Ryb25SZW5kZXJlciUyMCUzRCUyMCh0eXBlb2YlMjBwcm9jZXNzJTIwISUzRCUzRCUyMCd1bmRlZmluZWQnJTIwJTI2JTI2JTIwdHlwZW9mJTIwcHJvY2Vzcy52ZXJzaW9ucyUyMCElM0QlM0QlMjAndW5kZWZpbmVkJyUyMCUyNiUyNiUyMHR5cGVvZiUyMHByb2Nlc3MudmVyc2lvbnMuZWxlY3Ryb24lMjAhJTNEJTNEJTIwJ3VuZGVmaW5lZCclMjAlMjYlMjYlMjBwcm9jZXNzLnR5cGUlMjAlM0QlM0QlM0QlMjAncmVuZGVyZXInKSUzQiU1Q24lMkYlMkYlMjBPUyUyMGRldGVjdGlvbiU1Q25pZiUyMCh0eXBlb2YlMjBuYXZpZ2F0b3IlMjAlM0QlM0QlM0QlMjAnb2JqZWN0JyUyMCUyNiUyNiUyMCFpc0VsZWN0cm9uUmVuZGVyZXIpJTIwJTdCJTVDbiUyMCUyMCUyMCUyMF91c2VyQWdlbnQlMjAlM0QlMjBuYXZpZ2F0b3IudXNlckFnZW50JTNCJTVDbiUyMCUyMCUyMCUyMF9pc1dpbmRvd3MlMjAlM0QlMjBfdXNlckFnZW50LmluZGV4T2YoJ1dpbmRvd3MnKSUyMCUzRSUzRCUyMDAlM0IlNUNuJTIwJTIwJTIwJTIwX2lzTWFjaW50b3NoJTIwJTNEJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdNYWNpbnRvc2gnKSUyMCUzRSUzRCUyMDAlM0IlNUNuJTIwJTIwJTIwJTIwX2lzSU9TJTIwJTNEJTIwKF91c2VyQWdlbnQuaW5kZXhPZignTWFjaW50b3NoJyklMjAlM0UlM0QlMjAwJTIwJTdDJTdDJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdpUGFkJyklMjAlM0UlM0QlMjAwJTIwJTdDJTdDJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdpUGhvbmUnKSUyMCUzRSUzRCUyMDApJTIwJTI2JTI2JTIwISFuYXZpZ2F0b3IubWF4VG91Y2hQb2ludHMlMjAlMjYlMjYlMjBuYXZpZ2F0b3IubWF4VG91Y2hQb2ludHMlMjAlM0UlMjAwJTNCJTVDbiUyMCUyMCUyMCUyMF9pc0xpbnV4JTIwJTNEJTIwX3VzZXJBZ2VudC5pbmRleE9mKCdMaW51eCcpJTIwJTNFJTNEJTIwMCUzQiU1Q24lMjAlMjAlMjAlMjBfaXNXZWIlMjAlM0QlMjB0cnVlJTNCJTVDbiUyMCUyMCUyMCUyMF9sb2NhbGUlMjAlM0QlMjBuYXZpZ2F0b3IubGFuZ3VhZ2UlM0IlNUNuJTIwJTIwJTIwJTIwX2xhbmd1YWdlJTIwJTNEJTIwX2xvY2FsZSUzQiU1Q24lN0QlNUNuZWxzZSUyMGlmJTIwKHR5cGVvZiUyMHByb2Nlc3MlMjAlM0QlM0QlM0QlMjAnb2JqZWN0JyklMjAlN0IlNUNuJTIwJTIwJTIwJTIwX2lzV2luZG93cyUyMCUzRCUyMChwcm9jZXNzLnBsYXRmb3JtJTIwJTNEJTNEJTNEJTIwJ3dpbjMyJyklM0IlNUNuJTIwJTIwJTIwJTIwX2lzTWFjaW50b3NoJTIwJTNEJTIwKHByb2Nlc3MucGxhdGZvcm0lMjAlM0QlM0QlM0QlMjAnZGFyd2luJyklM0IlNUNuJTIwJTIwJTIwJTIwX2lzTGludXglMjAlM0QlMjAocHJvY2Vzcy5wbGF0Zm9ybSUyMCUzRCUzRCUzRCUyMCdsaW51eCcpJTNCJTVDbiUyMCUyMCUyMCUyMF9sb2NhbGUlMjAlM0QlMjBMQU5HVUFHRV9ERUZBVUxUJTNCJTVDbiUyMCUyMCUyMCUyMF9sYW5ndWFnZSUyMCUzRCUyMExBTkdVQUdFX0RFRkFVTFQlM0IlMjAlMjAlMjAlMjAlNUNuJTIwJTIwJTIwJTIwX2lzTmF0aXZlJTIwJTNEJTIwdHJ1ZSUzQiU1Q24lN0QlNUNubGV0JTIwX3BsYXRmb3JtJTIwJTNEJTIwMCUyMCUyRiolMjBXZWIlMjAqJTJGJTNCJTVDbmlmJTIwKF9pc01hY2ludG9zaCklMjAlN0IlNUNuJTIwJTIwJTIwJTIwX3BsYXRmb3JtJTIwJTNEJTIwMSUyMCUyRiolMjBNYWMlMjAqJTJGJTNCJTVDbiU3RCU1Q25lbHNlJTIwaWYlMjAoX2lzV2luZG93cyklMjAlN0IlNUNuJTIwJTIwJTIwJTIwX3BsYXRmb3JtJTIwJTNEJTIwMyUyMCUyRiolMjBXaW5kb3dzJTIwKiUyRiUzQiU1Q24lN0QlNUNuZWxzZSUyMGlmJTIwKF9pc0xpbnV4KSUyMCU3QiU1Q24lMjAlMjAlMjAlMjBfcGxhdGZvcm0lMjAlM0QlMjAyJTIwJTJGKiUyMExpbnV4JTIwKiUyRiUzQiU1Q24lN0QlNUNuZXhwb3J0JTIwY29uc3QlMjBpc1dpbmRvd3MlMjAlM0QlMjBfaXNXaW5kb3dzJTNCJTVDbmV4cG9ydCUyMGNvbnN0JTIwaXNNYWNpbnRvc2glMjAlM0QlMjBfaXNNYWNpbnRvc2glM0IlNUNuZXhwb3J0JTIwY29uc3QlMjBpc0xpbnV4JTIwJTNEJTIwX2lzTGludXglM0IlNUNuZXhwb3J0JTIwY29uc3QlMjBpc05hdGl2ZSUyMCUzRCUyMF9pc05hdGl2ZSUzQiU1Q25leHBvcnQlMjBjb25zdCUyMGlzV2ViJTIwJTNEJTIwX2lzV2ViJTNCJTVDbmV4cG9ydCUyMGNvbnN0JTIwaXNJT1MlMjAlM0QlMjBfaXNJT1MlM0IlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIydW1kJTIyJTJDJTIybmFtZSUyMiUzQSUyMm15QnVuZGxlJTIyJTJDJTIyYW1kJTIyJTNBJTdCJTIyaWQlMjIlM0ElMjIlMjIlN0QlMkMlMjJnbG9iYWxzJTIyJTNBJTdCJTdEJTdEJTJDJTIyZXhhbXBsZSUyMiUzQW51bGwlN0Q=) might be slightly different, but in my real project it produces the following code:

```javascript
else if (typeof process === 'object') {
        _isWindows = (process.platform === 'win32');
        _isMacintosh = (process.platform === 'darwin');
        process.platform === 'linux');  // <--- here 
        _locale = LANGUAGE_DEFAULT;
        _language = LANGUAGE_DEFAULT;
...
```
notice that the line is missing an opening parenthesis and variable assignment

**Comment by lukastaegert** [comment]
Thanks! I will investigate.

**Comment by lukastaegert** [comment]
Fix at #3926
